scripts/make-folders.sh

```diff
@@ -1,35 +1,35 @@
 #!/usr/bin/env bash
 
 # Make folders for all the days in the calendar
-for i in {01,02,03,04,05,06,07,08,09,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25}
+for i in {10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25}
 do
+    DISPLAY_DAY="Day $i"
     DAY="days/day-$i"
     rm -r "./$DAY"
     mkdir -p "./$DAY/solutions/"
-    cat > "./$DAY/solutions/.keep" << KEEP
-KEEP
+    touch "./$DAY/solutions/.keep"
+
+    mkdir -p "./$DAY/io/"
+    touch "./$DAY/io/.keep"
 
     cat > "./$DAY/README.md" << README
 # $DAY
 ## --- Part 1 ---
```

Comment:
I'm pretty sure most of the readme fiiles we have so far have these two combined.

It could be changed to this:

`# ---  $DAY: Title ---`

Not  that that really matters in the template, as it's going to be written over at some point again...
¯\\\_(ツ)\_/¯

Reply:
I think it matters, even if it is a minor thing, because the template is supposed to help the developer. It is a way of documenting things, and communicating intent, so that any contributor is capable of contributing README.md's. I will fix in a commit directly. :+1: